mlir/lib/Bindings/Python/IRCore.cpp

```diff
@@ -770,7 +846,7 @@ void PyMlirContext::clearOperationAndInside(PyOperationBase &op) {
   MlirOperationWalkCallback invalidatingCallback = [](MlirOperation op,
                                                       void *userData) {
     PyMlirContextRef &contextRef = *static_cast<PyMlirContextRef *>(userData);
-    contextRef->clearOperation(op);
+    contextRef->clearOperationLocked(op);
```

Comment:
maybe i'm missing something but i don't see an explicit `nb::ft_lock_guard lock(liveOperationsMutex);` anywhere around here?

Reply:
Renamed the method.